winbuild/build_prepare.py

```diff
@@ -689,6 +702,11 @@ def build_dep_all() -> None:
         disabled += ["libimagequant"]
     if args.no_fribidi:
         disabled += ["fribidi"]
+    elif args.architecture == "ARM64" and platform.machine() != "ARM64":
+        print(
+            "\nWarning: Cross-compiling FriBiDi is currently not supported, disabling"
+        )
```

Comment:
You didn't want to use `warnings.warn()`?

Reply:
I don't like that it can sometimes appear at a random point in the output due to using stderr instead of stdout so I hadn't even tested it. I've just tried it and found that:
* in Terminal, it seems to always print in the intended order;
* in PyCharm, it can print out of order, but uses a red highlight.

I've therefore changed the code to use `warnings.warn`.